### ISSUE TITLE
Commands not working at all

### ISSUE THREAD

**Issue by Bristingr** [foxbot/Discord.Addons.Interactive#19]
After applying the sample code to try it out and see how it works, none of the interactive commands seem to work.  Even creating a new program and doing a straight copy-paste (changing the bot key of course) does nothing.

Does this feature work with the latest version of .Net or am I missing something?

**Comment by JanVargovsky** [comment]
Yes, it is working. I'm using the library with .Net Core 2.1, 3.0 and 3.1, everything is fine. Can you try `SampleApp`?

**Comment by anf** [comment]
> After applying the sample code to try it out and see how it works, none of the interactive commands seem to work. Even creating a new program and doing a straight copy-paste (changing the bot key of course) does nothing.

@Bristingr, does it work now? If so, you can close this issue 😅
